object_tracker/object_tracker.py

```diff
@@ -26,7 +26,11 @@ def run_frame(self, img):
         orig = img.copy()
         optical_flow_output = None
         if self._run_optical_flow:
-            optical_flow_output = self._match(orig)
+            try:
```

Comment:
dimp (tracker,run_frame) will never fail/return none?

Reply:
I see it may return none in case of input=none, and as far as i understand in the current implementation it may happen

Reply:
i add a check for None return